rudder-webapp/debian/rules

```diff
@@ -120,8 +120,8 @@ binary-arch: install
 	dh_install --SOURCEDIR=$(CURDIR)/SOURCES/rudder-doc pdf /usr/share/doc/rudder
 	dh_install --SOURCEDIR=$(CURDIR)/SOURCES/rudder-doc html /usr/share/doc/rudder
 
-	# Install SELinux policies
-	dh_install --SOURCEDIR=$(CURDIR)/SOURCES/ rudder-webapp.pp /opt/rudder/share/selinux/
+	# Install SELinux policies -> Not done yet, only on RHEL
+	# dh_install --SOURCEDIR=$(CURDIR)/SOURCES/ rudder-webapp.pp /opt/rudder/share/selinux/
```

Comment:
We completely removed SELinux support from Debian packages at the moment. Please remove this instead of commenting it, no dead code :)